ONSdigital/design-system
# Update GA attributes on modals

### What is the context of this PR?
This change is to add the functionality of pushing to the GA datalayer when the modal opens and closes including click, button press and timeout.

It also:
- Removes the initialised GA attributes on modals and their tests as these aren't used
- Tidies up some code
- Solves an issue where there is two elements of the same id on a page

### How to review
- Check initialised attributes are not added
- Check open and closed attributes are set
- Set the `enableGA` param on the example modal and check the console when the modal opens and closes to see the data that would be pushed to the datalayer
